akkeeper/src/main/scala/akkeeper/master/route/ContainerController.scala

```diff
@@ -23,8 +23,7 @@ import akkeeper.api._
 import akkeeper.common.{ContainerDefinition, ContainerDefinitionJsonProtocol}
 import scala.concurrent.ExecutionContext
 
-class ContainerController(service: ActorRef)(implicit dispatcher: ExecutionContext,
-                                             timeout: Timeout)
+class ContainerController(val service: ActorRef)(implicit timeout: Timeout)
```

Comment:
Specifying a constructor argument like `val service: ...` make this argument public. I don't think that reference to `service` should be publicly available, since it's not a part of the interface and is too implementation specific,

Reply:
Done